apps/researcher/src/app/[locale]/communities/[slug]/buttons.tsx

```diff
@@ -77,86 +74,20 @@ export function JoinCommunityButton({communityId, communitySlug}: Props) {
   );
 }
 
-const menuItems = [
-  {translationKey: 'settingsButton', page: 'settings'},
-  {translationKey: 'membersButton', page: 'members'},
-];
-
-interface ButtonGroupProps {
-  slideOutEditFormId: string;
-  communityId: string;
-  communitySlug: string;
-}
-
-export function ButtonGroup({
-  slideOutEditFormId,
-  communityId,
-  communitySlug,
-}: ButtonGroupProps) {
+export function ManageMembersButton({communityId, communitySlug}: Props) {
+  const {openProfile} = useCommunityProfile({communitySlug, communityId});
   const t = useTranslations('Community');
-  const {openOrganizationProfile} = useClerk();
-  const {setActive} = useOrganizationList();
 
-  const openClerkProfile = (firstPage: string) => {
-    // Only the active organization can be edited.
-    setActive && setActive({organization: communityId});
-    // We want to show only one page of the organization's profile.
-    // But it is impossible to load only one page, so the next best thing is to hide the navbar and only show the first page.
-    // We must place all pages in `customPage` to define the page order.
-    // Pages not in `customPages` will load before the custom pages. So, we need to add all pages to control the first loaded page.
-    openOrganizationProfile({
-      afterLeaveOrganizationUrl: `/revalidate/?path=/[locale]/communities/${communitySlug}&redirect=/communities/${communitySlug}`,
-      customPages: ['settings', 'members']
-        .sort((a, b) => {
-          return a === firstPage ? -1 : b === firstPage ? 1 : 0;
-        })
-        .map(page => ({label: page})),
-      appearance: {
-        elements: {
-          navbar: 'hidden',
-        },
-      },
-    });
+  const manageMembersClick = () => {
```

Comment:
Would the condensed version also work, `const manageMembersClick = () => openProfile(...)`?